packages/ui-components/src/display/natValue/roundToDecimalPlaces.js

```diff
@@ -0,0 +1,20 @@
+// @ts-check
+import { assert, details } from '@agoric/assert';
+
+export const roundToDecimalPlaces = (
```

Comment:
Hmm, can you explain more? I like being able to name the file with its function rather than having a messy and long `helpers` file. I would probably prefer a `helpers` folder rather than file, if possible.

Reply:
Moved `roundToDecimals` and `captureNum` to a helpers folder.